xrootd/xrdproto/auth/krb5/krb5.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package krb5 contains the implementation of krb5 (Kerberos) security provider.
+package krb5
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	"go-hep.org/x/hep/xrootd/xrdproto/auth"
+	"gopkg.in/jcmturner/gokrb5.v5/client"
+	"gopkg.in/jcmturner/gokrb5.v5/config"
+	"gopkg.in/jcmturner/gokrb5.v5/credentials"
+	"gopkg.in/jcmturner/gokrb5.v5/crypto"
+	"gopkg.in/jcmturner/gokrb5.v5/messages"
+	"gopkg.in/jcmturner/gokrb5.v5/types"
+)
+
+// Default is a Kerberos 5 client configured from cached credentials.
+// If the credentials could not be correctly configured, Default will be nil.
+var Default auth.Auther
```

Comment:
`s/auth.Auther/*Auth/`
(so `Default.Provider()` will work even if `Default` is `nil`.)

Reply:
In that case during:
```go
providers := []auth.Auther{krb5.Default, unix.Default}
	for _, provider := range providers {
		if provider != nil {
                     //...
```
`provider` is always not `nil` since there is an assignment from pointer to the interface. :-/

Reply:
right. ok, le'ts leave `var Default auth.Auther` then :)